scripts/bench/tts-bench.py

```
@@ -0,0 +1,650 @@
+    lines.append("")
+    lines.append("| engine | utterance | lang | audio s | cold s | median s | RTF | x real-time | peak RSS MB |")
+    lines.append("|---|---|---|---|---|---|---|---|---|")
+    for engine in report["engines"]:
+        if engine.get("skipped"):
+            continue
+        for row in engine["rows"]:
+            lines.append(
+                f"| {engine['name']} | {row['utterance']} | {row['lang']} | {row['audio_seconds']:.2f} | "
+                f"{row['cold_wall_seconds']:.2f} | {row['median_wall_seconds']:.2f} | "
+                f"{row['median_rtf']:.3f} | {row['median_x_realtime']:.1f}x | {row['peak_rss_kb'] / 1024:.0f} |"
+            )
```

Ruff (0.16.1)

[warning] Use `list.extend` to create a transformed list

(PERF401)